local_units/management/commands/import_local_units_health.py

```diff
@@ -45,7 +44,9 @@ def handle(self, *args, **options):
 
                 country = country_name_id_map[row['COUNTRY'].lower()]
                 type = local_unit_id_map[int(row['TYPE CODE'])]
-                is_public = row['VISIBILITY'].lower() == 'public'
+                visibility = 3 if row['VISIBILITY'].lower() == 'public' else 1
+                data_source_id = row['DATA SOURCE ID']
+                health_id = row['DATA SOURCE ID']
```

Comment:
Can't we just have a single field to store this information?

Reply:
Good idea!
My only reason was: the better visibility on Admim page. But ok, I'll keep only the unavoidable foreign key reference then.